packages/distribution/addon/components/cd-inquiry-dialog/inquiry-part.js

```diff
@@ -33,10 +33,7 @@ export default class CdInquiryDialogInquiryPartComponent extends Component {
 
     const inquiry = this.args.inquiry;
 
-    if (
-      inquiry.status === "SUSPENDED" &&
-      this.abilities.can("edit inquiry", inquiry)
```

Comment:
Why did you drop this?

Reply:
Because without edit permission it is difficult to distinguish whether an inquiry was already sent or not.

Reply:
Ah makes sense. I added this because I didn't want the creator of the inquiry to see this. But they only see the answer once it's completed - so NVM!